app/components/forms/wizard/basic-details-step.js

```diff
@@ -36,8 +36,8 @@ export default Component.extend(FormMixin, EventWizardMixin, {
     return orderBy(paymentCurrencies, 'name');
   }),
 
-  canAcceptPayPal: computed('data.event.paymentCurrency', 'settings.paypalBraintreeSandboxAccessToken', 'settings.paypalBraintreeAccessToken', function() {
-    return (this.get('settings.paypalBraintreeSandboxAccessToken') || this.get('settings.paypalBraintreeAccessToken')) && find(paymentCurrencies, ['code', this.get('data.event.paymentCurrency')]).paypal;
+  canAcceptPayPal: computed('data.event.paymentCurrency', 'settings.paypalSandboxClient', 'settings.paypalClient', function() {
+    return (this.get('settings.paypalSandboxClient') || this.get('settings.paypalClient')) && find(paymentCurrencies, ['code', this.get('data.event.paymentCurrency')]).paypal;
```

Comment:
Make use of the fields added in https://github.com/fossasia/open-event-server/pull/5188